cmd/access/node_builder/access_node_builder.go

```diff
@@ -737,7 +742,25 @@ func (builder *FlowAccessNodeBuilder) BuildExecutionSyncComponents() *FlowAccess
 				execDataDistributor.AddOnExecutionDataReceivedConsumer(builder.ExecutionIndexer.OnExecutionData)
 
 				return builder.ExecutionIndexer, nil
-			}, indexerDependencies)
+			}, indexerDependencies).
+			Component("script execution", func(node *cmd.NodeConfig) (module.ReadyDoneAware, error) {
```

Comment:
I'm not sure if this will get initialized only after the ingestion is done cc @peterargue

Reply:
no, since the indexers is a `DependableComponent`, anything that depends on it must be too. This doesn't really look like a component either since it doesn't implement the `ReadyDoneAware` interface. You could probably just run this code at the end of the indexer closure, and we can figure out a better place for it later when integrating it into the access api